addons/mail/tests/common.py

```diff
@@ -501,6 +501,21 @@ def _find_mail_mail_wid(self, mail_id, status=None, mail_message=None, author=No
             )
         return mail
 
+    def _find_mail_mail_wpartners_or_emails(self, recipients, mail_tos, status, mail_message=None, author=None, content=None, email_from=None):
```

Comment:
Not a fan for this, as we hide a lot of stuff in the helper itself. Can't we stay with other lower-level tools ? I fear that each layer add possibilities that we are not testing what we want / what we expect ...